programs/relayer/src/core/builder.rs

```diff
@@ -55,6 +54,13 @@ impl RelayerBuilder {
         tracing::info!(%socket_addr, "Starting relayer...");
         let socket_addr = socket_addr.parse::<std::net::SocketAddr>()?;
 
+        let (mut health_reporter, health_service) = tonic_health::server::health_reporter();
+
+        let reflection_service = tonic_reflection::server::Builder::configure()
+            .register_encoded_file_descriptor_set(tonic_health::pb::FILE_DESCRIPTOR_SET)
+            .register_encoded_file_descriptor_set(api::FILE_DESCRIPTOR_SET)
+            .build_v1()?;
```

Comment:
what does the reflection service allow us to do?

Reply:
allows to describe protos without actually having them, it's useful when you're doing local debugging, e.g. `grpcurl localhost:3000 describe relayer.RelayerService` allows you to see what methods you can call. alternatively, grpc-ui automatically creates a UI for calling all of the methods without needing the proto files.